How to implement External Profile Data for Engage V16. (#7453)
umbraco/UmbracoDocs: 16/umbraco-engage/developers/profiling/external-profile-data.md

### Register custom components

> To render this External Profile Tab with a custom component, you have to create your component and register it with Umbraco Engage. The following code will show how to do both. Add the below code in a Typescript file:

Reviewer: ```suggestion
To render this External Profile Tab with a custom component, create your component and register it with Umbraco Engage. The following code demonstrates both steps. Add the below code in a TypeScript file:
```
